lib/kernel/doc/src/application.xml

```diff
@@ -237,6 +237,37 @@ Nodes = [cp1@cave, {cp2@cave, cp3@cave}]</code>
           keys, respectively.</p>
       </desc>
     </func>
+    <func>
+      <name name="set_env" arity="1" since="OTP 22.0"/>
```

Comment:
Please change the value of the since attribute to "OTP \@OTP-15642\@". OTP-15642 is the internal ticket number for this feature, and it will be substituted with the correct OTP release number when this is actually released.